x-pack/test/security_solution_endpoint_api_int/apis/metadata.ts

```diff
@@ -35,8 +35,13 @@ import { TRANSFORM_STATES } from '../../../plugins/security_solution/common/cons
 
 export default function ({ getService }: FtrProviderContext) {
   const supertest = getService('supertest');
+  const endpointTestResources = getService('endpointTestResources');
 
   describe('test metadata apis', () => {
+    before(async () => {
+      await endpointTestResources.setMetadataTransformFrequency('1s');
```

Comment:
ideally we'd set the united transform to 1s as well but as of 8.0.0, we get 403s when trying to edit the united transform. likely related to fleet permission changes since it utilizes the fleet-agents index.